tools/clang/lib/Headers/hlsl/vk/spirv.h

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2024 Google LLC
+//
+// This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef _HLSL_VK_SPIRV_H_
+#define _HLSL_VK_SPIRV_H_
+
+namespace vk {
+
+enum StorageClass {
+  StorageClassWorkgroup = 4,
+};
```

Comment:
why not pull the entire non-11 C++ spirv-headers like we do?

Reply:
This is something we are working on for those headers in general:
 - how to distribute them, and where shall they live
 - shall we add a new flavor of spirv headers files like we have for c++11 and non-11
We do plan on bringing the whole spirv-header file, this is more a temporary measure until we figure out the whole thing.

Reply:
This is the start of the standard headers, and I am working on adding cooperative matrices as an interesting proof of concept. I am doing the minimum I need to in order to do that. In the near future, we will decide how we want to handle the core functionality going forward.